web-server/v0.4/src/services/explore.js

```diff
@@ -19,3 +19,35 @@ export default async function querySharedSessions(params) {
     },
   });
 }
+
+/* The updateDescription function here take the edited value of description from the user 
+and updates it in the database. */
+
+/* Later after updating the database, we are using querySharedSessions method which is referred as 
```

Comment:
This comment contains references to files which can change over time.  I'd refran from referring to the external file structure in comments.

Instead, consider dropping this entirely and instead adding a comment to `querySharedSessions` about what it does.

Reply:
Updated single line comments without any file reference.